<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Passwordl</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="logUser_input">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Login_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>If you are new User </string>
    </property>
   </widget>
   <widget class="QPushButton" name="registration_button">
    <property name="geometry">
     <rect>
      <x>619</x>
      <y>440</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">margin-right:auto;
margin-left:auto;
</string>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="userName_label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="logPass_input">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="fishIcon">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/images/fish-logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fish Buddies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-300</x>
      <y>-100</y>
      <width>3671</width>
      <height>1216</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width:100%;
height:100%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/images/wind-wave-cartoon-water-waves-6e912dfb99f8506682a88de1e14af64b.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="logWarning_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : red
</string>
    </property>
    <property name="text">
     <string>Username and password do not match </string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>submit_button</zorder>
   <zorder>password_label</zorder>
   <zorder>logUser_input</zorder>
   <zorder>Login_label</zorder>
   <zorder>registration_button</zorder>
   <zorder>userName_label</zorder>
   <zorder>logPass_input</zorder>
   <zorder>fishIcon</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>logWarning_label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
